web/crux/src/services/kratos.service.ts

```diff
@@ -117,4 +117,29 @@ export default class KratosService {
     })
     return req.data
   }
+
+  async enableOnboarding(userId: string): Promise<void> {
```

Comment:
What is the motivation to created 2 HTTP API for this?

Reply:
So this way there is no need for a body. it's also more restful IMHO. For example, Spotify uses the same PUT / DELETE on a `:songId/star` resource when you save a song to your liked songs list.